exercises/secret-handshake/introduction.md

```diff
@@ -0,0 +1,12 @@
+# Introduction
+
+You are making a secret coding club with some friends and friends-of-friends.
+You don't know everyone in the club, so you and your friends have decided to create a secret handshake that you can use to recognise that someone is a member.
+You don't want anyone who isn't in the know to be able to crack the code.
+
+The secret handshake is created by taking a binary number and turning it into a series of actions.
```

Comment:
"is created by taking a binary number"

Maybe rephrase slightly to make it seem more, eh, designed? As in: you and friends have designed a binary coding scheme or something like that.